appveyor.yml

```diff
@@ -4,7 +4,7 @@ image:
 nuget:
   account_feed: true
 
-version: 7.11.0-rc.{build}
+version: 7.11.0.{build}
```

Comment:
Should be 7.11.0
Please remove '.{build}'

Reply:
ok